lib/src/exceptions.dart

```diff
@@ -18,27 +16,26 @@ AuthException convertJsAuthError(interop.AuthError jsError) {
 }
 
 AuthException convertJsClientConfigurationErrorMessage(
-    interop.ClientConfigurationErrorMessage jsError) {
-  return ClientConfigurationMessageException._jsObject(jsError);
-}
+        interop.ClientConfigurationErrorMessage jsError) =>
+    ClientConfigurationMessageException._jsObject(jsError);
```

Comment:
This is ok, but did a formatter do this? Personally, I'd prefer to keep the bracket syntax since it doesn't fit on one line.